src/types/lang.rs

```diff
@@ -0,0 +1,148 @@
+  GermanATNew ("New Austrian") => "naustrian",
+  GermanCHNew ("New Swiss German") => "nswissgerman",
+  Greek ("Greek") => "greek",
+  Hungarian ("Hungarian") => "magyar" | "hungarian",
+  Icelandic ("Icelandic") => "icelandic",
+  Italian ("Italian") => "italian",
+  Latvian ("Latvian") => "latvian",
+  Lithuanian ("Lithuanian") => "lithuanian",
+  Marathi ("Marathi") => "marathi",
+  NorwegianBokmal ("Norwegian (Bokmål)") => "norsk",
+  NorwegainNynorsk ("Norwegian (Nynorsk)") => "nynorsk",
+  Polish ("Polish") => "polish",
+  PortugueseBR ("Brazilian Portuguese") => "brazil",
+  Portuguese ("Portuguese") => "portuguese" | "portuges",
+  Romanian ("Romanian") => "romanian",
+  Russian ("Russian") => "russian",
+  SerbianLatin ("Serbian (Latin)") => "serbian",
+  SerbianCyrillic ("Serbian (Cyrillic)") => "serbianc",
+  Slovak ("Slovak") => "slovak",
+  Slovene ("Slovene") => "slovene" | "slovanian",
+  Spanish ("Spanish") => "spanish",
+  Swedish ("Swedish") => "swedish",
+  Turkish ("Turkish") => "turkish",
+  Ukranian ("Ukranian") => "ukranian"
+}
+
+#[cfg(feature = "unic-langid")]
+impl From<Language> for LanguageIdentifier {
+    fn from(value: Language) -> Self {
+        match value {
+            Language::Basque => langid!("eu"),
+            Language::Bulgarian => langid!("bg"),
+            Language::Catalan => langid!("ca"),
+            Language::Croatian => langid!("hr"),
+            Language::Czech => langid!("cs"),
+            Language::Danish => langid!("da"),
+            Language::Dutch => langid!("nl"),
+            Language::EnglishUS => langid!("en-US"),
+            Language::EnglishUK => langid!("en-UK"),
+            Language::EnglishAUS => langid!("en-AU"),
+            Language::EnglishNZ => langid!("en-NZ"),
+            Language::Estonian => langid!("et"),
+            Language::Finnish => langid!("fi"),
+            Language::French => langid!("fr"),
+            Language::German | Language::GermanNew => langid!("de"),
+            Language::GermanAT | Language::GermanATNew => langid!("de-AT"),
+            Language::GermanCH | Language::GermanCHNew => langid!("de-CH"),
+            Language::Greek => langid!("el"),
+            Language::Hungarian => langid!("hu"),
+            Language::Icelandic => langid!("is"),
+            Language::Italian => langid!("it"),
+            Language::Latvian => langid!("lv"),
+            Language::Lithuanian => langid!("lt"),
+            Language::Marathi => langid!("mr"),
+            Language::NorwegianBokmal => langid!("nb"),
+            Language::NorwegainNynorsk => langid!("nn"),
+            Language::Polish => langid!("pl"),
+            Language::PortugueseBR => langid!("pt-BR"),
+            Language::Portuguese => langid!("pt-PT"),
+            Language::Romanian => langid!("ro"),
+            Language::Russian => langid!("ru"),
+            Language::SerbianLatin => langid!("sr-LATIN"),
+            Language::SerbianCyrillic => langid!("sr-CYRILLIC"),
```

Comment:
Not sure if there is some special handling by this package, but these seem to be the correct identifiers according to this utility https://util.unicode.org/UnicodeJsps/languageid.jsp :
```suggestion
            Language::SerbianLatin => langid!("sr-Latn"),
            Language::SerbianCyrillic => langid!("sr-Cyrl"),
```